### ISSUE TITLE
Can't load the plug page

### ISSUE THREAD

**Issue by shuaishuairuirui** [scipion-em/scipion-pyworkflow#258]

**Comment by pconesa** [comment]
Hi @shuaishuairuirui could you be more specific?

I guess you are talking about the plugin manager?

Do you see any error in the console?

**Comment by shuaishuairuirui** [comment]
> 
> 
> Hi @shuaishuairuirui could you be more specific?
> 
> I guess you are talking about the plugin manager?
> 
> Do you see any error in the console?

yes,when I installed scipion.And I open the plugin manager.The page always loading.I can get the plugin menu.

**Comment by shuaishuairuirui** [comment]
![J IW(}8RLBRB93WJD~B6{CO](https://user-images.githubusercontent.com/59869039/120775843-2a272580-c556-11eb-860f-547596e49ff4.png)

**Comment by pconesa** [comment]
Do you have an internet connection from that machine?

Either pypi.org server was down when you tried or your machine. 

I'm trying now, and I'm not getting an error, but is taking too long to load! There might be something with pypi maybe.

**Comment by shuaishuairuirui** [comment]
Yes.
My machine has internet connection.And I can ping pypi.org.So I don't know why I have this problem.I also loading a very long time.But it still did't work.

When I first install scipion.I can load the plugin.But when I using Xmipp-heighresoulutio It takes a longtime still didn't using the GPU.So I think maybe something wrong in intalltion.So I reinstall the scipion.Then I cant load the plugin.Is there some err?Thanks

shuaishuai

**Comment by shuaishuairuirui** [comment]
![Screenshot from 2021-06-05 11-42-54](https://user-images.githubusercontent.com/59869039/120879041-24cbe880-c5f3-11eb-8f32-f6564acac5bd.png)
I don't know my process is running?Because my memory seems did using many,and my GPU seems didn't using.My GPU card is RTX8000.Could you give my some suggestion about that.

**Comment by pconesa** [comment]
Has the plugin manager issue been fixed?

I'll assign this to a xmipp developer

**Comment by shuaishuairuirui** [comment]
> Has the plugin manager issue been fixed?
> 
> I'll assign this to a xmipp developer

When I open the plugin manager overnight.I can loading the plugin.But I still need a halfday to loading the plugin again when I close the windows.Maybe something wrong?

**Comment by shuaishuairuirui** [comment]
![Screenshot from 2021-06-05 17-15-27](https://user-images.githubusercontent.com/59869039/120886728-faddea80-c621-11eb-84f0-4611df279291.png)
I follow the tutorial to processing.When I come to the cl2d.It seems didn't run for a long time?My particles have downsample toe <3A/pix.And only a few particles.But it seems didn't response?Could you give me some helps?Thanks very much.

**Comment by DStrelak** [comment]
Hi @shuaishuairuirui 

These are two issues, so I'm going to change this ticket and create another one for the CL2D.

**Comment by DStrelak** [comment]
@fonsecareyna82 , 
To me, this seems like a sequence of timeouts on all connections to Pypi we do via plugin manager.
Could we catch the timeout exception and report to user that there's a connection issue, asking him to check his connection or try later?

Notice that `ping` command is not equivalent to the requests we're doing (i.e you can have working ping and our requests still failing), as `ping` uses ICMP protocol, and we ask for TCP request on a specific port.
